cms/djangoapps/contentstore/views/item.py

```diff
@@ -266,6 +269,8 @@ def service(self, block, service_name):
                 return DjangoXBlockUserService(self._user)
             if service_name == "studio_user_permissions":
                 return StudioPermissionsService(self._user)
+            if service_name == "lti-configuration":
+                return ConfigurationService(LTIConsumerHideFieldsFlag)
```

Comment:
@muzaffaryousaf: Good question. @andy-armstrong might be a better person to ask (or perhaps @cahrens).